controls/pcidss_4_ocp4.yml

```diff
@@ -0,0 +1,3591 @@
+        it will be required and must be fully considered during a PCI DSS assessment.
+      levels:
+        - base
+      status: pending
+
+  - id: A1.2
+    title: Multi-tenant service providers facilitate logging and incident response for all
+      customers.
+    levels:
+      - base
+    status: pending
+    controls:
+    - id: A1.2.1
+      title: Audit log capability is enabled for each customer's environment that is consistent
+        with PCI DSS Requirement 10.
+      levels:
+        - base
+      status: pending
+
+    - id: A1.2.2
+      title: Processes or mechanisms are implemented to support and/or facilitate prompt forensic
+        investigations in the event of a suspected or confirmed security incident for any
+        customer.
+      description: |-
+        Forensic investigation is readily available to all customers in the event of a suspected
+        or confirmed security incident.
+      levels:
+        - base
+      status: pending
+
+    - id: A1.2.3
+      title: Processes or mechanisms are implemented for reporting and addressing suspected or
+        confirmed security incidents and vulnerabilities.
+      levels:
+        - base
+      status: pending
+
+  - id: A2.1
+    title: POI terminals using SSL and/or early TLS are confirmed as not susceptible to known
+      SSL/TLS exploits.
+    description: |-
+      Appendix A2: Additional PCI DSS Requirements for Entities Using SSL/Early TLS for
+      Card-Present POS POI Terminal Connections.
+    levels:
+      - base
+    status: pending
+    controls:
+    - id: A2.1.1
+      title: Where POS POI terminals at the merchant or payment acceptance location use SSL and/or
+        early TLS, the entity confirms the devices are not susceptible to any known exploits for
+        those protocols.
+      levels:
+        - base
+      status: pending
+      notes: |-
+        Related to requirements 2.2.7 and 3.5.1.1.
+        Service level settings for web servers such as Apache and NGINX should also be checked.
+
+    - id: A2.1.2
+      title: 'Additional requirement for service providers only: All service providers with
+        existing connection points to POS POI terminals that use SSL and/or early TLS as defined
+        in A2.1 have a formal Risk Mitigation and Migration Plan in place.'
+      levels:
+        - base
+      status: pending
+
+    - id: A2.1.3
+      title: 'Additional requirement for service providers only: All service providers provide a
+        secure service offering.'
+      description: |-
+        This requirement is not eligible for the customized approach. This requirement applies
+        only when the entity being assessed is a service provider.
+      levels:
+        - base
+      status: pending
+
+  - id: A3.1
+    title: A PCI DSS compliance program is implemented.
+    description: |-
+      Appendix A3: Designated Entities Supplemental Validation (DESV)
+    levels:
+      - base
```

Comment:
In the PCI-DSS 3.2.1 profile the Annex A3.1 was assigned level `desv`.

Reply:
These levels are only applicable to the build system, right? From what I understand, there isn't a way to invoke these or consume these levels through the operator directly?

Reply:
At the moment these levels are only used by the build system to select specific rules.
But as the control file gets get more exposure, these levels become more visible and accessible.

I'm okay dropping them as they are not clearly defined in PCI-DSS policy.